packages/roosterjs-content-model-core/lib/editor/core/createEditorCore.ts

```diff
@@ -18,6 +20,21 @@ import type {
  */
 export function createEditorCore(contentDiv: HTMLDivElement, options: EditorOptions): EditorCore {
     const corePlugins = createEditorCorePlugins(options, contentDiv);
+    const plugins = (options.plugins ?? []).filter(x => !!x);
+    const domToModelOptions: DomToModelOption[] = [];
+    const modelToDomOptions: ModelToDomOption[] = [];
+
+    plugins.forEach(x => {
+        const config = x.getContentModelConfig?.();
+
+        if (config?.domToModelOption) {
+            domToModelOptions.push(config.domToModelOption);
```

Comment:
If two plugins override the same element handler (such as image), both of the handlers will take effect on the element?

Reply:
If that really happens, the second one wins

Reply:
Here in ImageEditPlugin I only add an additional format parser and applier, I won't really override handlers.

Reply:
should we make these additional parses and applier coexist? Otherwise, a plugin parser/applier can break a functionality of the plugin.

Reply:
We don't hard request them coexist. Sometimes we only need one of them, although here we need both.